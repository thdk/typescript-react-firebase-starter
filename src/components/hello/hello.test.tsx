import * as React from "react";

import { render, cleanup } from "@testing-library/react";
import { Hello } from "./";

afterEach(cleanup);

it("renders", () => {
    const { asFragment } = render(<Hello compiler="compiler" framework="framework" />);
    expect(asFragment()).toMatchSnapshot();
});
