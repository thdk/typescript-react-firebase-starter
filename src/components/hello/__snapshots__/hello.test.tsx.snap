// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <h1>
    Hello from 
    <span
      class="red"
    >
      compiler
    </span>
     and 
    <span
      class="green"
    >
      framework
    </span>
    !
  </h1>
</DocumentFragment>
`;
